embeddings/pipeline/lightning_pipeline.py

```diff
@@ -0,0 +1,30 @@
+from typing import Generic, TypeVar
+
+from embeddings.data.datamodule import BaseDataModule, Data
+from embeddings.evaluator.evaluator import Evaluator
+from embeddings.model.model import Model
+from embeddings.pipeline.pipeline import Pipeline
+
+EvaluationResult = TypeVar("EvaluationResult")
+ModelResult = TypeVar("ModelResult")
+
+
+class LightningPipeline(
+    Pipeline[EvaluationResult],
+    Generic[Data, ModelResult, EvaluationResult],
```

Comment:
we can add TODO either an issue how do we want to unify pipeline components in the future comparing this to flair-based

Reply:
Added to #120